drivers/SmartThings/philips-hue/src/disco/button.lua

```diff
@@ -63,7 +63,7 @@ local function _do_update(api_instance, device_service_info, bridge_id, resource
   end
 
   if type(cache) == "table" then
-    cache[resource_id] = button_remote_description
+    cache[button_remote_description.id] = button_remote_description
```

Comment:
will `button_remove_description.id` always be nil? it seems like it, but maybe this is being set on line 49?